R/datasets.R

```diff
@@ -597,4 +597,24 @@
 #' \item{year}{Year of Sale}
 #' \item{region}{Region Where Sale Took Place}
 #' }
-"avocados"
+
+
+
+#' House Prices and Properties in Saratoga, New York
+#' Gathered from <https://docs.google.com/spreadsheets/d/1AY5eECqNIggKpYF3kYzJQBIuuOdkiclFhbjAmY3Yc8E/edit#gid=622599674>
+#' 
+#' @format A data frame with 1057 observations on the following 8 variables
+#' \describe{
+#' \item{Price}{Price (US dollars)} 
+#' \item{Living.Area}{Living Area (square feet)}
+#' \item{Bathrooms}{Number of Bathroom (half bathrooms have no shower or tub)}
+#' \item{Bedrooms}{Number of Bedrooms}
+#' \item{Fireplaces}{Number of Fireplaces}
+#' \item{Lot.Size}{Size of Lot (acres)}
+#' \item{Age}{Age of House (years)}
+#' \item{Fireplace}{Whether the house has a Fireplace}
+#' }
+#' @source random sample of 1057 houses taken from full Saratoga Housing Data (De Veaux)
+"saratoga_ny_home_prices" 
```

Comment:
This needs to match the name of the data set. So it should be `"saratoga_houses"`.